### DISCUSSION TITLE
Issue with Windows titlebar after Deepin 20.9

### DISCUSSION THREAD

**Discussion by Olyno** [linuxdeepin/developer-center#4126] [Troubleshooting | 故障排查] [answered]
Hi 👋🏻 
Since version 20.9 of Deepin, the titlebar is far too small in my opinion. It is not very accessible for people with vision difficulties.

Would it be possible to correct this problem? By reverting this change, or by adding an option in the settings to configure the size of the titlebar?

Related too:

> Optimization of the height of the window tube titlebar from 40pixels to 24pixels

![See image here: https://cdn.discordapp.com/attachments/856354115474948106/1098310732464013312/image.png](https://cdn.discordapp.com/attachments/856354115474948106/1098310732464013312/image.png)

**Comment by BLumia** [comment]
This change is because too many users complain about the titlebar is too height. It might be hard to find a value that everyone is happy with, but luckily you can use `deepin-tweak` to tweak the titlebar height value. :)

**Reply by Olyno** [reply]
That would be great! Is it installed by default? Or do I have to install it manually? And what would be the associated command to put in the terminal (if there is one)? Thanks a lot for your help in any case!

**Reply by BLumia** [reply, marked as answer]
Simply `sudo apt install deepin-tweak`, and you can find it from the launcher or run it by executing `deepin-tweak`.

**Reply by Olyno** [reply]
It's weird, no matter what value I put, I don't see anything that changes. I can put 200, 40 or any other value, it does not change the size of the titlebar.

I'll go to the repository concerned directly. Thanks for your help!

**Comment by raspin0** [comment]
I believe this change is a regression. It looks much worse and less coherent with the rest of the desktop. It's less convenient too.

**Reply by BLumia** [reply]
Not a regression bug since it's intended (it's even in the release note). I personally prefer a shorter titlebar for non-CSD window and a lots of people doesn't like the original height (see https://github.com/linuxdeepin/developer-center/issues/1210). The `deepin-tweak` tool can be used if you are not happy with the default value. (if it's not working since there is a related bug currently, you can track the bug here: https://github.com/linuxdeepin/developer-center/issues/4157)

**Reply by raspin0** [reply]
I've received 3 separate complaints from users about this issue already.

**Reply by BLumia** [reply]
Yeah, we can't make everyone happy with one single default value. See #1210 to know how many people complaint about it before we change the height back to a shorter value 😂
